transformer_engine/common/include/transformer_engine/fused_attn.h

```diff
@@ -78,12 +84,14 @@ enum NVTE_Mask_Type {
  *  - O = D * V.T
  *
  * Support Matrix:
- *  | precision |    qkv layout   |  bias   |  mask   | sequence length |  head_dim  |
- *  |    FP8    | QKV_INTERLEAVED | NO_BIAS | PADDING |   <= 512        |      64    |
+ *  | precision |    qkv layout   |  bias                   |  mask          | sequence length |  head_dim  |
+ *  |    FP8    | QKV_INTERLEAVED | NO_BIAS                 | PADDING        |   <= 512        |      64    |
+ *  | FP16/BF16 | QKV_INTERLEAVED | NO_BIAS/POST_SCALE_BIAS | PADDING/CAUSAL |   <= 512        |      64    |
```

Comment:
Please add a colum regarding the dropout support, i.e. Yes for FP8, No for BF16/FP16 as of now. Thanks!

Reply:
Added in https://github.com/NVIDIA/TransformerEngine/pull/175/commits/d717f535ea0c9c50ef426c6419182791ea9e9092